src/pages/use-cases/code-security.tsx

```diff
@@ -343,15 +341,18 @@ const UseCasePage: FunctionComponent = () => (
         <div className="bg-light-gray-3 py-7">
             <ContentSection>
                 <div className="row d-flex flex-column mx-4 mx-lg-0 align-items-lg-center align-items-left">
-                    <div className="mb-5 d-flex flex-column">
+                    <div className="mb-5 d-flex flex-column text-start text-md-center mx-auto align-items-lg-center">
                         <h1 className="font-weight-bold">Get started with Sourcegraph</h1>
-                        <p>Find, fix, and track vulnerable code quickly across your entire codebase.</p>
+                        <p className="max-w-450">
+                            Find, fix, and track vulnerable code quickly across your entire codebase to improve code
+                            security.
+                        </p>
                     </div>
-                    <div className="d-flex flex-column">
+                    <div className="text-center col-12 px-0">
                         <Link href="/demo" passHref={true}>
                             {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
                             <a
-                                className="btn btn-primary"
+                                className="btn btn-primary min-w-200"
```

Comment:
Maybe we should loop in Tracey here to get her opinion. I think all these buttons should take up screen width on mobile, especially since the same button in the hero does this. When we have a standardized style, maybe we should make a re-usable component for this section used on all the Use Cases to ensure they stay standardized. It would help us require tracking too, if we require it as a prop + if we do that sort of approach on our commonly used CTA's.
(Although a new `max-w` class on the hero buttons is giving us a weird UI bug there too heh)
![image](https://user-images.githubusercontent.com/59381432/172266749-a01413c9-bcdf-4ee4-ba41-b008f80b39aa.png)
![image](https://user-images.githubusercontent.com/59381432/172266788-875792bd-5a3e-494c-9b2a-ebdb8e56fb96.png)

Reply:
I don't think we should use min-width almost anywhere throughout the site unless there's a really good use for it. When we use it, we should override it at the next below or above breakpoint. But in most cases we should use max-width which is more efficient for responsive flexibility and reduces the amount of code for breakpoint specific width overrides.

If we want to span the buttons a little bit bigger on mobile, I think we should only go up to a max of 350px wide, otherwise it's just too big IMO.

@zlonko this should give you what you're looking for to match @st0nebraker's first screenshot, or `max-w-350` for the latter.

```suggestion
                                className="btn btn-primary max-w-200 w-100"
```

Either way, I agree that we should have some consistency throughout the site but let's scope that in another ticket to avoid scope creep here. Same thing regarding a common/shared button component. We could potentially create a Button component with a `variation` prop to consolidate a mapping of a few different CTAs we have (ie: `demo`, `try`, etc.) but I wouldn't want to overcomplicate it since not all button and link CTAs are the same.

Reply:
The `Request demo` buttons spacing should still match each other while we wait for any brand design changes. That's what the spec has outlined [here](https://www.figma.com/file/kQTq5eWvjbMcMmsvZ8wvyF/Code-reuse?node-id=106%3A53). If the buttons width has a `max-w-350` I think it should be centered to the screen. I agree if it's scope creep a follow up ticket can be made for this. Thoughts?

Reply:
@st0nebraker It looks like I misunderstood, although I am encountering some trouble with the link due to a Figma outage. To clarify: 
- Both Request Demo buttons should have a max-width of 350
- Both Request Demo buttons should be centered to the screen on mobile

Is the screenshot what you are looking for on mobile? Please let me know what I may be missing. I can then apply these changes to the other pages. 

<img width="300" alt="Screen Shot 2022-06-07 at 11 48 58 AM" src="https://user-images.githubusercontent.com/60713139/172438283-a17e7575-5637-443a-bd6d-881969446a44.png">

<img width="300" alt="Screen Shot 2022-06-07 at 11 52 55 AM" src="https://user-images.githubusercontent.com/60713139/172438973-5b6bddea-f073-46b0-8dc4-cb1e38aeab49.png">

<img width="300" alt="Screen Shot 2022-06-07 at 11 44 25 AM" src="https://user-images.githubusercontent.com/60713139/172437340-cad823bd-f3a2-412d-af7f-4867dc080a28.png">

Reply:
Yes that screenshot matches the spec! 👍🏻

Reply:
Great! Your suggestion is a great idea. I fixed the width and alignment of the Request Demo buttons on the other four use cases pages as well.

Reply:
Thank you! Looks good 😄